queue-load-leveling/src/main/java/org/queue/load/leveling/Message.java

```diff
@@ -0,0 +1,56 @@
+/**
+ * The MIT License
+ * Copyright (c) 2014 Ilkka Seppälä
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.queue.load.leveling;
+
+/**
+ *  Message class with only one parameter.
+ *
+*/
+public class Message {
+  private String msg;
+
+  // Empty constructor.
+  public Message() {
+  }
+  
+  // Parameter constructor.
+  public Message(String msg) {
+    super();
+    this.msg = msg;
+  }
+
+  // Get Method for attribute msg.
+  public String getMsg() {
+    return msg;
+  }
+  
+  // Set Method for attribute msg.
+  public void setMsg(String msg) {
+    this.msg = msg;
+  }
+  
+  @Override
+  public String toString() {
+    return msg;
+  }
+}
```

Comment:
I would refactor to make this class immutable. Remove no-args constructor and setter and make `msg` final.